src/main/js/store/actions/roles.js

```diff
@@ -89,75 +89,56 @@ export function fetchRoles() {
                 'X-CSRF-TOKEN': csrfToken,
                 'Content-Type': 'application/json'
             }
-        }).then((response) => {
-            if (response.ok) {
-                response.json().then((jsonArray) => {
-                    dispatch(fetchedAllRoles(jsonArray));
-                });
-            } else {
-                switch (response.status) {
-                    case 401:
-                    case 403:
-                        dispatch(verifyLoginByStatus(response.status));
-                        break;
-                    default:
-                        response.json().then((json) => {
-                            let message = '';
-                            if (json && json.message) {
-                                // This is here to ensure the message is a string. We have gotten UI errors because it is somehow an object sometimes
-                                message = json.message.toString();
-                            }
-                            dispatch(fetchingAllRolesError(message));
+        })
+            .then((response) => {
```

Comment:
I think having the lines not starting with . makes more sense. My IDE was doing this to me automatically with the save actions. Perhaps you are having the same issue?